src/lib.rs

```diff
@@ -774,16 +760,18 @@ mod tests {
     use super::*;
 
     use futures::StreamExt;
-    use slog::Drain;
-    use snafu::Whatever;
+    use tracing::Level;
+
+    fn init() {
```

Comment:
suggestion: Make it more clear _what_ the function inits:

```suggestion
    fn init_tracing_subscriber() {
```